### PR TITLE
remove the tmp fix

### PR DESCRIPTION
**Pull request by kay0u**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
         "email": "cyp@rouquin.me"
     },
     "requirements": {
-        "yunohost": ">= 4.1.0"
+        "yunohost": ">= 4.2.8"
```
Comment (alexAubin) [inline manifest.json:17]:
```suggestion
        "yunohost": ">= 4.2.7"
```

**Review by alexAubin — COMMENTED** [review]
Tmp fix because zblerg the CI doesn't have 4.2.8 yet

**Comment by alexAubin** [comment]
Upgrades are failing because trying to request the app redirects on the SSO ... probably a permission issue

**Comment by kay0u** [comment]
!testme

**Pull request merged**